packages/venia-ui/lib/components/Accordion/accordion.js

```diff
@@ -0,0 +1,41 @@
+import React, { createContext, useContext } from 'react';
+
+import { useAccordion } from '@magento/peregrine/lib/talons/Accordion/useAccordion';
+
+import { mergeClasses } from '../../classify';
+import defaultClasses from './accordion.css';
+
+// To be controlled by the corresponding talon.
+let contextValue = {
+    handleSectionToggle: () => {},
+    lastSectionId: '',
+    openSections: new Set([])
+};
+
+const AccordionContext = createContext(contextValue);
+const { Provider } = AccordionContext;
+
+const Accordion = props => {
+    const { canOpenMultiple = true, children } = props;
+
+    // The talon is the source of truth for the context value.
+    const talonProps = useAccordion({ canOpenMultiple, children });
+    const { handleSectionToggle, lastSectionId, openSections } = talonProps;
+    contextValue = {
```

Comment:
```suggestion
    const contextValue = { handleSelectionToggle, openSections }
```

---
I see what you've done here, but I'd suggest just creating the `const` here. If you need to set some default values like you're doing above, I think you should do that in the talon.

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2090/commits/83fecaaaaaaf3326967ed88ee1c6fc641bf1aef1, good call.